### PR TITLE
typos in readme

### PR DESCRIPTION
**Pull request by dilpath**
Logo was linked to a deleted feature branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 ![CI](https://github.com/yannikschaelte/yaml2sbml/workflows/CI/badge.svg)
 [![codecov](https://codecov.io/gh/yannikschaelte/yaml2sbml/branch/master/graph/badge.svg?token=AORT3GREQJ)](https://codecov.io/gh/yannikschaelte/yaml2sbml)
```
Comment (dilpath) [inline README.md:5]: Is it intended that these links are to forks or previous locations of the repository? Unable to select all links in this "review" comment, there's one more 1 line above.

@martamatos @yannikschaelte

Comment (dilpath) [reply README.md:5]: There are also some `utm` parameters that maybe could be removed.

File: README.md [hunk 2]
```diff
@@ -82,7 +78,7 @@ For instance, using the yaml file in the examples folder:
  yaml2sbml ../examples/ode_input1.yaml ../examples/sbml_out.xml
 ```
 
-#### yaml2PEtab command
+#### yaml2petab command
```
Comment (dilpath) [inline README.md:81]: Not sure, but as SBML is capitalized but the tool is `yaml2sbml`, then should `yaml2petab` be lowercase here too @jvanhoefer ?

Comment (yannikschaelte) [reply README.md:81]: Valid point. I would prefer to lowercase this, also in the corresponding functions. makes typing easier and is more consistent ;)

File: README.md (hunk 1 above)
Comment (yannikschaelte) [reply README.md:5]: I should have updated the old links to the new location in https://github.com/yaml2sbml-dev/yaml2sbml/pull/81/files already.

File: README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
 ![CI](https://github.com/yannikschaelte/yaml2sbml/workflows/CI/badge.svg)
 [![codecov](https://codecov.io/gh/yannikschaelte/yaml2sbml/branch/master/graph/badge.svg?token=AORT3GREQJ)](https://codecov.io/gh/yannikschaelte/yaml2sbml)
 
-<img src="https://raw.githubusercontent.com/yaml2sbml-dev/yaml2sbml/feature_logo/doc/logo/logo_yaml2sbml_long.png" width="80%" alt="yaml2sbml logo"/>
+![yaml2sbml logo](doc/logo/logo_yaml2sbml_long.png)
```
Comment (yannikschaelte) [inline README.md:7]: The problem is that we copy the README.md onto pypi as well, and there the relative path will break. Thus I think we need the absolute URL.

**Review by yannikschaelte — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (yannikschaelte) [reply README.md:7]: Fully agreed though that this was the wrong link. I think one should replace feature_logo by master and then quickly merge things there. Also needs to be done in doc/index.rst in #81

File: README.md (hunk 1 above)
Comment (jvanhoefer) [reply README.md:5]: Nope, some links in there are obsolete. Marta and me had a discussion in #55 and decided to merge the rewrite of the README after the "you can pip install the package is done". Will do this today :)

Comment (dilpath) [reply README.md:5]: Will leave as-is for this PR then

File: README.md (hunk 3 above)
Comment (dilpath) [reply README.md:9]:
```suggestion
![yaml2sbml logo](https://github.com/yaml2sbml-dev/yaml2sbml/blob/master/doc/logo/logo_yaml2sbml_long.png?raw=true)
```

**Review by jvanhoefer — APPROVED** [review]
:+1:

**Review by yannikschaelte — APPROVED** [review]

**Pull request merged**